### PR TITLE
Small bunch of changes to make examples compileable.

### PR DESCRIPTION
**Pull request by Altai-man**
It includes such types of fixes:
1)Missed semicolons;
2)Typos;
3)Endless code without :skip-test;
4)Convert flags table to actual table, not code.

Review is welcome.

### CODE REVIEW

**Comment by jonathanstowe** [comment]
Seems fairly uncontroversial to me :)

File: doc/Type/Int.pod6 [hunk 1]
```diff
@@ -36,8 +36,9 @@ respectively:
 
 These notations allow you to use variables, too:
 
-    :16($ninety-nine) # 153
-    :100[99, $two, 3] # 990203
+    my Int $ininety-nine = 99;
+    :16($ninety-nine); # 153
+    :100[99, $two, 3]; # 990203
```
Comment (AlexDaniel) [inline doc/Type/Int.pod6]: but what is $two?

File: doc/Type/WrapHandle.pod6 [hunk 2]
```diff
@@ -8,8 +8,8 @@ class WrapHandle { ... }
 
 C<WrapHandle> is created and returned by L<wrap|/type/Routine#method_wrap>. It's only use is to unwrap wrapped routines. Either call L<unwrap|/type/Routine#method_unwrap> on a routine object or call the method C<restore> on a C<WrapHandle> object.
 
-    sub f(){ say 'f was called' }
-    my $wrap-handle = &f.wrap({ say 'before'; callsame; say 'after' })
+    sub f(){ say 'f was called' };
```
Comment (AlexDaniel) [inline doc/Type/WrapHandle.pod6]: a semicolon after ``sub f(){ }`` ? Why?

File: doc/Type/Int.pod6 (hunk 1 above)
Comment (Altai-man) [inline doc/Type/Int.pod6]: Should be added too! Thanks.

File: doc/Type/WrapHandle.pod6 (hunk 2 above)
Comment (Altai-man) [inline doc/Type/WrapHandle.pod6]: Because we can't automatically compile and check this example otherwise?

File: doc/Type/Int.pod6 (hunk 1 above)
Comment (tommystanton) [inline doc/Type/Int.pod6]: Looks like there's a small typo introduced in the first line addition here (`$ininety-nine`).

**Pull request merged**